### PR TITLE
Changing name for new tools

### PR DESCRIPTION
**Pull request by saumilhaptik**

### CODE REVIEW

**Review by medhaanand — APPROVED** [review]

**Review by medhaanand — APPROVED** [review]

File: website/static/js/index.js [hunk 1]
```diff
@@ -21,10 +21,14 @@ function getTitle() {
...
 
 document.addEventListener('DOMContentLoaded', function () {
-  const title = getTitle();
+  title = getTitle();
 
   if (title === 'Docs_haptik') return;
 
+  if (title === 'Bot Builder') title = 'Conversation Studio';
+  if (title === 'Agent Chat') title = 'Smart Agent Chat';
+  if (title === 'Bot Analytics') title = 'Intelligent Analytics';
```
Comment (RandiveSushant) [inline website/static/js/index.js:30]: Will URLs in Docusaurus change automatically or will that require a programattic change?  @medhaanand

Comment (medhaanand) [reply website/static/js/index.js:30]: This is not for changing the URL, it is for the title which comes on top of the page after you go inside any section.

Comment (RandiveSushant) [reply website/static/js/index.js:30]: Yes, I get that. What this function is doing is, taking the URL and changing its case(lower/upper) to make it a proper looking title. that is why I asked if the URLs are also changing then will we need this code?

File: docs/admin-tools/permission-management.md [hunk 2]
```diff
@@ -18,26 +18,26 @@ A Partner Admin is a special role that can be assigned to someone who needs to m
...
+* **Conversation Studio**
 
-    * **Bot Builder View** -  This role allows you to view a bot's flows and training data.
-    * **Bot Builder Edit** - This role allows you to edit a bot's flows and training data.
-    * **Bot Builder Transfer** - This role allows you to transfer a bot to production.
-    * **Bot Builder Business Manager** - This role allows you to manage the business configurations and also transfer the business to production.
+    * **Conversation Studio View** -  This role allows you to view a bot's flows and training data.
+    * **Conversation Studio Edit** - This role allows you to edit a bot's flows and training data.
+    * **Conversation Studio Transfer** - This role allows you to transfer a bot to production.
+    * **Conversation Studio Business Manager** - This role allows you to manage the business configurations and also transfer the business to production.
```
Comment (GanadiniAkshay) [inline docs/admin-tools/permission-management.md:26]: Permission management names are in the next sprint

Comment (medhaanand) [reply docs/admin-tools/permission-management.md]: Reverted these changes

**Pull request merged**